src/snapred/backend/recipe/algorithm/SaveGroupingDefinition.py

```diff
@@ -123,6 +128,10 @@ def PyExec(self) -> None:
             f"Cleanup the zero calibration workspace {cal_ws_name}",
             Workspace=cal_ws_name,
         )
+        self.mantidSnapper.WashDishes(
+            f"Cleanup the grouping workspace {grouping_ws_name}",
+            Workspace=grouping_ws_name,
+        )
```

Comment:
This _could_ point to the user's grouping workspace, in which case we'd wipe it.  This should check how the grouping workspace was set first,